packages/ui/avatar/avatarSystem.ts

```diff
@@ -108,9 +123,11 @@ export class AvatarEntity extends Entity {
   }
 
   setExpression(id: string, timestamp: number): void {
-    const shape = this.avatarShape
-    shape.expressionTriggerId = id
-    shape.expressionTriggerTimestamp = timestamp
+    setTimeout(() => {
```

Comment:
this would delay _all_ emotes by 1 sec.

Plus, there is not setTimeout available for the scenes once we go full-sandboxed mode